### PR TITLE
Notification system updates

### PR DESCRIPTION
**Pull request by salman-codewar**
- Added actionable notifications types
- Notifications resourceId bug fix
- Subscriber notification flow issue fixes

### CODE REVIEW

File: systems/common/events/event.go [hunk 1]
```diff
@@ -274,15 +274,15 @@ var EventToEventConfig = map[EventId]EventConfig{
...
+		Type:        notif.TYPE_ACTIONABLE_INFO,
 	},
 	EventMeshNodeOffline: {
 		Key:         EventMeshNodeOffline,
 		Name:        "EventMeshNodeOffline",
 		Title:       "Mesh Node Offline",
 		Description: "Mesh Node Offline",
 		Scope:       notif.SCOPE_ORG,
-		Type:        TypeDefault,
+		Type:        notif.TYPE_ACTIONABLE_INFO,
```
Comment (Brackleycassinga) [inline systems/common/events/event.go:285]: Why is "node offline" something we need to act on? I don't think it needs any action. It should just be TYPE INFO .

File: systems/common/notification/notification.go [hunk 2]
```diff
@@ -35,11 +35,15 @@ func (l NotificationScope) Value() (driver.Value, error) {
 type NotificationType uint8
 
 const (
-	TYPE_INAVLID  NotificationType = 0
-	TYPE_INFO     NotificationType = 1
-	TYPE_WARNING  NotificationType = 2
-	TYPE_ERROR    NotificationType = 3
-	TYPE_CRITICAL NotificationType = 4
+	TYPE_INAVLID             NotificationType = 0
```
Comment (Brackleycassinga) [inline systems/common/notification/notification.go:38]: There is a typo here it should be `INVALID`.

**Review by Brackleycassinga — APPROVED** [review]

File: systems/common/events/event.go (hunk 1 above)
Comment (salman-codewar) [reply systems/common/events/event.go:285]: This kind of notification supposed to lead to detail page. So in this case, if `ABC` node is offline that an actionable info which will lead to `ABC` notification detail page where use can perform curtain type of actions. like `restart`.

File: systems/common/notification/notification.go (hunk 2 above)
Comment (salman-codewar) [reply systems/common/notification/notification.go]: addressed

File: systems/common/rest/client/subscriber/subscriber.go [hunk 3]
```diff
@@ -20,7 +20,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-const SubscriberEndpoint = "/v1/subscribers"
+const SubscriberEndpoint = "/v1/subscriber"
```
Comment (delvatt) [inline systems/common/rest/client/subscriber/subscriber.go:23]: Why are we changing this?

**Review by delvatt — APPROVED** [review]

File: systems/common/rest/client/subscriber/subscriber.go (hunk 3 above)
Comment (salman-codewar) [reply systems/common/rest/client/subscriber/subscriber.go:23]: it should be `subscriber`, that was a typo.

Comment (delvatt) [reply systems/common/rest/client/subscriber/subscriber.go:23]: Normally speaking, rest resources names inside endpoints should be plural.  We need to update this in subscriber system api-gtw

**Pull request merged**